source/configuration/modules/imfile.rst

```diff
@@ -396,7 +406,7 @@ Note: parameter names are case-insensitive.
 
    **Default: unset**
 
-   **This paramater is deprecated.** It still is accepted, but should
+   **This parameter is deprecated.** It still is accepted, but should
```

Comment:
We probably should warn against the use. A common error in those days where it was necessary was that folks copy-and-pasted the def, ending up with dupe state file names, which caused (and I thinks still causes) rsyslog to get totally confused. To avoid this, we now have the automatic name generation.

Maybe an extra section at the end for deprecated attributes?

Reply:
> Maybe an extra section at the end for deprecated attributes?

That makes sense. Here though, I was just fixing a typo that my editor flagged.

Reply:
yeah, I used the opportunity ;-)